.github/workflows/scripts/gendependabot.sh

```diff
@@ -51,7 +51,7 @@ get_pip_ignores() {
 
     ignore:
       - dependency-name: "urllib3"
-        versions: [">=2"]
+        versions: [">=3"]
```

Comment:
Why are other dep requirements saying `urllib<3` but this is `>=3`?

Reply:
This is to ignore updates for 3 and above